Automattic/vip-governance-plugin
# Use the new block editing mode for locking

## Description

- This will take advantage of the new `useBlockEditingMode()` feature added in WordPress 6.4 - https://make.wordpress.org/core/2023/10/17/miscellaneous-editor-changes-in-wordpress-6-4/#new-public-hook-in-the-block-editor-api-useblockeditingmode. It only does this if the method exists, which would be in WordPress 6.4.
- Update the packages, including removing lodash as we don't use it anymore. As a result of updating the linting package, we need to ensure that the linting is done correctly so this has been updated as well.

Fixes #44, #64, #58.

## Steps to Test

- Load up a WP env with 6.4 and 6.3, using this plugin.
- Setup a post with some disabled blocks.
- 6.4:  Make sure they don't show up in the sidebar
- 6.3: They should show up in the sidebar still
